plugin-gradle/src/main/java/com/diffplug/gradle/spotless/FormatExtension.java

```diff
@@ -185,11 +185,23 @@ protected FileCollection parseTarget(Object target) {
 		}
 	}
 
+	private static void relativizeIfSubdir(List<String> relativePaths, File root, File dest) {
+		String relativized = relativize(root, dest);
+		if (relativized != null) {
+			relativePaths.add(relativized);
+		}
+	}
+
+	/**
+	 * Returns the relative path between root and dest,
+	 * or null if dest is not a child of root.
+	 */
+	@Nullable
```

Comment:
Sweet!  That's where I wanted it to be, but I thought Java didn't let me do that.  Turns out it does! Fixed in the next commit, once it passes CI I'll merge so that @regrog can use 3.5.0-SNAPSHOT.